src/main/resources/python/run_tests.py

```
@@ -29,56 +83,64 @@ def addFailure(self, test, err):
     try:
         if not os.path.exists(test_dir):
-            print(f"##teamcity[message text='Test directory not found: {test_dir}' status='ERROR']")
+            tc_print('message', text=f"Test directory not found: {test_dir}", status='ERROR')
             sys.exit(1)
 
         suite = loader.discover(test_dir)
-
         runner = TeamCityTestRunner(stream=sys.stdout, verbosity=2)
         result = runner.run(suite)
 
     except Exception as e:
```

Ruff (0.14.10)

Do not catch blind exception: `Exception`

(BLE001)